### PR TITLE
Feature/unit tests

### PR DESCRIPTION
**Pull request by mjkoo**
Flesh out unit test suite

### CODE REVIEW

File: tests/test_corpus.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    pass
+
+    #d = DownloadToFileSystemCorpus()
+    #d.download_and_unpack()
+
+    #assert d.get_malicious_file_list()
+    #assert d.get_benign_file_list()
+
+    #d.generate_truth()
+    #assert os.path.exists(d.truth_db_pth)
```
Comment (ceseale) [inline tests/test_corpus.py]: Should these still be commented out?

**Comment by ceseale** [comment]
Great job! Look good overall :+1:

File: tests/test_corpus.py (hunk 1 above)
Comment (mjkoo) [reply tests/test_corpus.py]: Yeah don't want unit tests depending on external services, have a separate ticket about corpus stuff will add in "Restore test cases but with mocked responses" to that todo list

Comment (zv) [reply tests/test_corpus.py]: I think @mjkoo has it on the nose, no unit tests should communicate with a remote service.

I moved these outside of `corpus` and got them running only to get started on the unit test work (they were the only ones in the project at the time).

**Pull request merged**